Library/Homebrew/cask/lib/hbc/cli/search.rb

```diff
@@ -1,53 +0,0 @@
-require "search"
-
-module Hbc
-  class CLI
-    class Search < AbstractCommand
```

Comment:
What does `brew cask search` do now? Would be good to have a `odeprecated` message telling people to use `brew search`?

Reply:
There is, it's in `compat`.

Reply:
@reitermarkus Gotcha, thanks.